hbp_knowledge/receptors/dani2007.bel

```diff
@@ -57,96 +57,140 @@ DEFINE ANNOTATION TextLocation  AS URL "https://arty.scai.fraunhofer.de/artifact
+UNSET Confidence
 
 SET Evidence = "By modulating activity-dependent events, nAChRs participate in fundamental aspects of synaptic plasticity that are involved in attention, learning, memory, and development (3, 12–16)."
+SET Confidence = "High"
 p(FPLX:CHRN) -- bp(GO:"regulation of synaptic plasticity")
 p(FPLX:CHRN) -- path(MESH:Attention)
 p(FPLX:CHRN) -- bp(GO:learning)
 p(FPLX:CHRN) -- bp(GO:memory)
 p(FPLX:CHRN) -- bp(GO:"developmental process")
+UNSET Confidence
 
 SET Evidence = "Decline, disruption, or alterations of nicotinic cholinergic mechanisms have been implicated in various dysfunctions, such as schizophrenia, epilepsy, autism, Alzheimer’s disease (AD), and addiction (17–23)."
+SET Confidence = "High"
 p(FPLX:CHRN) -- path(MESH:Schizophrenia)
 p(FPLX:CHRN) -- path(MESH:Epilepsy)
 p(FPLX:CHRN) -- path(MESH:"Autistic Disorder")
 p(FPLX:CHRN) -- path(MESH:"Alzheimer Disease")
 p(FPLX:CHRN) -- path(MESH:"Behavior, Addictive")
+UNSET Confidence
 
 SET Evidence = "Mammalian nAChRs are cation selective, being permeable to small monovalent and divalent cations."
+SET Confidence = "High"
 p(FPLX:CHRN) -> tloc(a(MESH:"Cations, Monovalent"), MESH:"Extracellular Space", MESH:"Intracellular Space")
 p(FPLX:CHRN) -> tloc(a(MESH:"Cations, Monovalent"), MESH:"Intracellular Space", MESH:"Extracellular Space")
 p(FPLX:CHRN) -> tloc(a(MESH:"Cations, Divalent"), MESH:"Extracellular Space", MESH:"Intracellular Space")
 p(FPLX:CHRN) -> tloc(a(MESH:"Cations, Divalent"), MESH:"Intracellular Space", MESH:"Extracellular Space")
+UNSET Confidence
 
 SET Evidence = "Nicotinic receptor activity causes depolarization, and the divalent cation perme-
 ability plays an important physiological role by supplying ionic signals, including
 calcium (39–41)."
+SET Confidence = "High"
 act(p(FPLX:CHRN)) -> bp(GO:"membrane depolarization")
+UNSET Confidence
 
 SET Evidence = "Similarly, replacement of the α7 leucine at the synaptic, extracellular end (position 254 or 255) of the pore by threonine dramatically reduced the calcium permeability of the α7 receptors."
+SET Confidence = "High"
 p(HGNC:CHRNA7, sub(L,254,T)) -| tloc(a(CHEBI:"calcium(2+)"), MESH:"Extracellular Space", MESH:"Intracellular Space")
 p(HGNC:CHRNA7, sub(L,254,T)) -| tloc(a(CHEBI:"calcium(2+)"), MESH:"Intracellular Space", MESH:"Extracellular Space")
 p(HGNC:CHRNA7, sub(L,255,T)) -| tloc(a(CHEBI:"calcium(2+)"), MESH:"Extracellular Space", MESH:"Intracellular Space")
 p(HGNC:CHRNA7, sub(L,255,T)) -| tloc(a(CHEBI:"calcium(2+)"), MESH:"Intracellular Space", MESH:"Extracellular Space")
+UNSET Confidence
 
 SET Evidence = "Unlike many neurotransmitter signals that are shaped by pumps that return the transmitter to the intracellular space, the spread of ACh from the release site is determined by diffusion and by acetylcholinesterase (AChE) hydrolysis of ACh."
-pep(p(HGNC:ACHE)) => deg(a(CHEBI:acetylcholine))
+SET Confidence = "Medium"
+act(p(HGNC:ACHE), ma(pep)) => deg(a(CHEBI:acetylcholine))
```

Comment:
you can also give the peptidase activity in the form of the first statement. (BEL1.0)
So both statements mean exactly the same